share/picongpu/tests/PusherScaling/lib/python/test/Pusher/MainTest.py

```diff
@@ -0,0 +1,286 @@
+"""
+This file is part of PIConGPU.
+
+Copyright 2023 PIConGPU contributors
+Authors: Hannes Wolf
+License: GPLv3+
+
+Test script to determine if a Boris pusher works correctly. This is done by
+calculating the phase shift caused by the pusher.
+
+The phase shift is proportional to (dt)**2. The script checks whether the
+calculated exponent of (dt)**x is in the interval [1.9, 2.1] or not.
+The standard deviation should be < 0.05.
+
+"""
+
+import openpmd_api as io
+import numpy as np
+import sys
+
+
+class compare:
```

Comment:
Conventionally, class names should be PascalCase and nouns.

Reply:
Comparison